src/EnergyPlus/DemandManager.cc

```diff
@@ -393,95 +392,97 @@ void GetDemandManagerListInput(EnergyPlusData &state)
+                                    cCurrentModuleObject + "=\"" + thisDemandMgrList.Name + "\" invalid value " +
+                                        state.dataIPShortCut->cAlphaFieldNames(6) + "=\"" + state.dataIPShortCut->cAlphaArgs(6) + "\" not found.");
                     ErrorsFound = true;
                 }
             }
 
             // Get DEMAND MANAGER Type and Name pairs
-            DemandManagerList(ListNum).NumOfManager = int((NumAlphas - 6) / 2.0);
+            thisDemandMgrList.NumOfManager = int((NumAlphas - 6) / 2.0);
 
-            if (DemandManagerList(ListNum).NumOfManager > 0) {
-                DemandManagerList(ListNum).Manager.allocate(DemandManagerList(ListNum).NumOfManager);
+            if (thisDemandMgrList.NumOfManager > 0) {
+                thisDemandMgrList.Manager.allocate(thisDemandMgrList.NumOfManager);
 
-                for (MgrNum = 1; MgrNum <= DemandManagerList(ListNum).NumOfManager; ++MgrNum) {
+                for (MgrNum = 1; MgrNum <= thisDemandMgrList.NumOfManager; ++MgrNum) {
 
+                    auto &thisManager = thisDemandMgrList.Manager(MgrNum);
                     // Validate DEMAND MANAGER Type
                     {
-                        auto const SELECT_CASE_var(AlphArray(MgrNum * 2 + 5));
+                        auto const SELECT_CASE_var(state.dataIPShortCut->cAlphaArgs(MgrNum * 2 + 5));
                         if ((SELECT_CASE_var == "DEMANDMANAGER:LIGHTS") || (SELECT_CASE_var == "DEMANDMANAGER:EXTERIORLIGHTS") ||
                             (SELECT_CASE_var == "DEMANDMANAGER:ELECTRICEQUIPMENT") || (SELECT_CASE_var == "DEMANDMANAGER:THERMOSTATS") ||
                             (SELECT_CASE_var == "DEMANDMANAGER:VENTILATION")) {
 
-                            DemandManagerList(ListNum).Manager(MgrNum) = UtilityRoutines::FindItemInList(AlphArray(MgrNum * 2 + 6), DemandMgr);
+                            thisManager = UtilityRoutines::FindItemInList(state.dataIPShortCut->cAlphaArgs(MgrNum * 2 + 6), DemandMgr);
 
-                            if (DemandManagerList(ListNum).Manager(MgrNum) == 0) {
+                            if (thisManager == 0) {
                                 ShowSevereError(state,
-                                                CurrentModuleObject + "=\"" + state.dataIPShortCut->cAlphaArgs(1) + "\" invalid " +
-                                                    state.dataIPShortCut->cAlphaFieldNames(MgrNum * 2 + 6) + "=\"" + AlphArray(MgrNum * 2 + 6) +
-                                                    "\" not found.");
+                                                cCurrentModuleObject + "=\"" + thisDemandMgrList.Name + "\" invalid " +
+                                                    state.dataIPShortCut->cAlphaFieldNames(MgrNum * 2 + 6) + "=\"" +
+                                                    state.dataIPShortCut->cAlphaArgs(MgrNum * 2 + 6) + "\" not found.");
```

Comment:
The variable `state.dataIPShortCut->cAlphaArgs(1)` was not defined in this function and incorrectly used in the place of  `AlphArray(1)`. This line was the cause for the wrong object name. Similar incorrect uses in this function has also been fixed.